pkg/cmd/cli/cli.go

```diff
@@ -292,7 +292,7 @@ func CommandFor(basename string) *cobra.Command {
 	case "kubectl":
 		cmd = NewCmdKubectl(basename, out)
 	default:
-		cmd = NewCommandCLI(basename, basename, in, out, errout)
+		cmd = NewCommandCLI("oc", "oc", in, out, errout)
```

Comment:
I think this is still not the right approach. This path is executed by all commands that are build from our repo, this includes `oc`, `openshift`, `oadm`. Make sure to check them all.